aptos-move/aptos-vm/src/errors.rs

```diff
@@ -14,28 +14,20 @@ use move_deps::{
 /// These errors are only expected from the module that is registered as the account module for the system.
 /// The prologue should not emit any other error codes or fail for any reason, doing so will result
 /// in the VM throwing an invariant violation
-pub const EACCOUNT_FROZEN: u64 = 1000; // sending account is frozen
```

Comment:
why deleting this? we may froze account in future.

Reply:
Err how would we freeze an account? Governance proposals to freeze accounts wouldn't make sense as you can always front run them and move funds out :)

Reply:
sanctioned account, money laundry account.... maybe, we loose the capability now. But we are very likely to use it in future.

Reply:
WE can add back the error code later if needed. But there'll be no central authority to freeze accounts

Reply:
I don't think we as devs should/would have the ability to freeze accounts, that's basically means we're not decentralized and subject to regulations